examples/manipulating_visualizing/plot_neurovault_meta_analysis.py

```diff
@@ -0,0 +1,99 @@
+"""
+NeuroVault meta-analysis of stop-go paradigm studies.
+=====================================================================
+
+This example shows how to download statistical maps from
+NeuroVault
+
+See :func:`nilearn.datasets.fetch_neurovault` documentation for more details.
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+from nilearn import datasets
+
+### Define filters ############################################################
+cfilts = []  # [lambda col: col['DOI'] is not None]
+imfilts = [lambda im: "stop signal" in (im.get('cognitive_paradigm_cogatlas') or "").lower(),
```

Comment:
> The choices are: try to normalize, and risk that we'll get things
> wrong, be agnostic and let the data be peculiar to work with, or
> convince @chrisfilo to normalize things more easily across the board. I
> have a couple of issues open on it: NeuroVault/NeuroVault#395
> NeuroVault/NeuroVault#396

The best option would be convincing @chrisfilo.

Please, pretty please with a cherry on top, @chrisfilo.